Scripts/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+xcodebuild clean test -workspace Rocket.Chat.xcworkspace -scheme Rocket.Chat -destination "platform=iOS Simulator,name=iPhone 7" -enableCodeCoverage YES | xcpretty
```

Comment:
What if we want to add separated process to test iPhone 7+ and SE? Can we just add another script call here?